x-pack/plugins/ml/common/types/annotations.ts

```diff
@@ -93,3 +111,27 @@ export function isAnnotations(arg: any): arg is Annotations {
   }
   return arg.every((d: Annotation) => isAnnotation(d));
 }
+
+export interface FieldToBucket {
```

Comment:
The interfaces defined here will be applicable for other types than annotations I'm sure, but till we hit that use case, probably makes sense to leave them defined in here. I had a quick look to see if I could find something similar in our existing types, and the Kibana data plugin, but couldn't find anything.